extensions/arc/src/ui/dashboards/postgres/postgresWorkerNodesParametersPage.ts

```diff
@@ -0,0 +1,80 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as azdata from 'azdata';
+import * as azdataExt from 'azdata-ext';
+import * as loc from '../../../localizedConstants';
+import { IconPathHelper } from '../../../constants';
+import { PostgresServerParametersPage } from './postgresServerParameters';
+import { PostgresModel } from '../../../models/postgresModel';
+
+export class PostgresWorkerNodesParametersPage extends PostgresServerParametersPage {
+	private readonly _azdataApi: azdataExt.IExtension;
+
+	constructor(protected modelView: azdata.ModelView, _postgresModel: PostgresModel) {
+		super(_postgresModel, modelView);
+		this._azdataApi = vscode.extensions.getExtension(azdataExt.extension.name)?.exports;
```

Comment:
This stuff is also duplicated - can you not move this into the base class as well?